blocks/kanban-board/src/components/board/card/editable-card-content/styles.module.scss

```diff
@@ -0,0 +1,11 @@
+.wrapper {
+  border: none;
```

Comment:
According to the design this should also have `font-weight: 500` right?

Reply:
Right, updated it here https://github.com/hashintel/hash/pull/2234/commits/46cf9eef97f0e42071947a4d219fe3f46ddd9754